src/index.html

```diff
@@ -1,12 +1,395 @@
 <!doctype html>
-<html lang="en">
+<html class="page" lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="viewport"
+      content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__content">
+
+    <header class="header">
+      <a
+        class="header__logo-link"
+        href="#home">
+        <img
+          src="images/logo.png"
+          alt="Moyo logo">
```

Comment:
I won't reject for this kind of issues, but still – pay attention, please
```suggestion
      <a
        class="header__logo-link"
        href="#home"
      >
        <img
          src="images/logo.png"
          alt="Moyo logo"
        >
```

Reply:
I have an issue with vs code ressting when i use sign > on new line, still cant find a proper way through it.